src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
-public class ColorSupplier {
+import java.util.Random;
+
+public class ColorSupplier extends Ball {
     public String getRandomColor() {
-        return null;
+        Colors[] colors = Colors.values();
+        int length = colors.length;
+        int random = new Random(). nextInt(length);
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/43448085-bdca-4c56-90a7-987a4e6fd6ed)

Reply:
not fixed